### ISSUE TITLE
Why the example code always fail to run???

### ISSUE THREAD

**Issue by AdaJass** [aio-libs/aiohttp_session#32]
You must feel frustrated, if you try a new thing but fail in the simplest way. I run this code in windows 32 system.
the code is the example you give.
`
import asyncio
import time
from aiohttp import web
from aiohttp_session import get_session, session_middleware
from aiohttp_session.cookie_storage import EncryptedCookieStorage

@asyncio.coroutine
def handler(request):
    session = yield from get_session(request)
    session['last_visit'] = time.time()
    return web.Response(body=b'OK')

@asyncio.coroutine
def init(loop):
    app = web.Application(middlewares=[session_middleware(
        EncryptedCookieStorage(b'1111'))])
    app.router.add_route('GET', '/', handler)
    srv = yield from loop.create_server(
        app.make_handler(), '0.0.0.0', 8080)
    return srv

loop = asyncio.get_event_loop()
loop.run_until_complete(init(loop))
try:
    loop.run_forever()
except KeyboardInterrupt:
    pass
`

**Comment by AdaJass** [comment]
Traceback (most recent call last):
  File "C:\Users\Administrator\Desktop\HotIO\app.py", line 23, in <module>
    loop.run_until_complete(init(loop))
  File "D:\Anaconda3\lib\asyncio\base_events.py", line 337, in run_until_complete
    return future.result()
  File "D:\Anaconda3\lib\asyncio\futures.py", line 274, in result
    raise self._exception
  File "D:\Anaconda3\lib\asyncio\tasks.py", line 239, in _step
    result = coro.send(None)
  File "C:\Users\Administrator\Desktop\HotIO\app.py", line 16, in init
    EncryptedCookieStorage(b'1111'))])
  File "D:\Anaconda3\lib\site-packages\aiohttp_session\cookie_storage.py", line 27, in __init__
    self._fernet = fernet.Fernet(secret_key)
  File "D:\Anaconda3\lib\site-packages\cryptography\fernet.py", line 37, in __init__
    "Fernet key must be 32 url-safe base64-encoded bytes."
ValueError: Fernet key must be 32 url-safe base64-encoded bytes.

im some kind of don't know what 'url-safe base64-cncoded bytes' is look like.

**Comment by asvetlov** [comment]
Please use `EncryptedCookieStorage(b'a'*32)` or any 32-length bytesting

**Comment by AdaJass** [comment]
i try to wrote session myself and done. i don't know how to delete the "session" immediately when the user just close browser. So i set an interval to check the session table. could you tell me any idea about it, just the idea without any detail.  I think the cookie  is no need to encrypt if just use for mark user ,, unless you use it to transfer message. 

e..the example said the key is 16 bytes,,,,,,,,,,,,and i try 16 bytes fail all the way.

**Comment by shicky** [comment]
I have updated working [demo code here.](https://github.com/shicky/aiohttp_session/blob/master/demo/main.py)

```python
# secret_key must be 32 url-safe` base64-encoded bytes
fernet_key = fernet.Fernet.generate_key()
secret_key = base64.urlsafe_b64decode(fernet_key)
```

**Comment by asvetlov** [comment]
Fixed by #34
